packages/support/lib/fs.js

```diff
@@ -45,15 +48,15 @@ const fs = {
    * Copies files _and entire directories_
    * @param {string} source - Source to copy
    * @param {string} destination - Destination to copy to
-   * @param {...any} args - Additional arguments to pass to `ncp`
+   * @param {ncp.Options} [opts] - Additional arguments to pass to `ncp`
```

Comment:
would that be a breaking change? are these args used anywhere else in appium org?

Reply:
no. `ncp` accepts four parameters, one of which is a callback which is promisified away.  `source`, `dest`, and `options`.  any further arguments are discarded, so it is not necessary to use a rest param here.